helm/hpcc/templates/_helpers.tpl

```diff
@@ -1894,6 +1894,7 @@ args:
       {{- end -}}
      {{- end -}}
      {{- $_ := set $local "dnsNames" (uniq $local.dnsNames ) -}}
+     {{- $_ := set $local "allDomains" (prepend (default list $issuer.alternativeDomains) $domain ) -}}
```

Comment:
Minor: This the default operator is more normally written as

($issuer.alternativeDomains | default list)

(I had to read some documentation to check the semantics...)